src/old/clock/index.tsx

```diff
@@ -19,11 +19,20 @@ export interface ClockProps {
   customStyle?: ClockTheme
 }
 
+interface TimeComponent {
+  type: string,
+  value: string
+}
+
 export interface ClockState {
-  currentTime: Array<{type: string, value: string}>
+  currentTime: Array<TimeComponent>
   date: Date
 }
 
+function IsDayPeriod (timeComponent: TimeComponent): boolean {
```

Comment:
is the first letter in uppercase intentional?

Reply:
ping @petemill

Reply:
Ah yes, hard to break old habits - this one for module-root functions naming syntax! Fixed...